0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -175,6 +180,31 @@ ProtocolData ::= SEQUENCE OF SEQUENCE {
 }
 ```
 
+## Authentication
+
+Before anything else, when a client connects to a server, it sends a special
+`Message` request. Its primary `protocolData` entry MUST have name `'auth'`
```

Comment:
and a content-type of application/octet-stream

Reply:
Hm, what is the type of empty content, though? 🤔 
I'm ok with picking octet-stream, I guess it feels nicer to pick one.

Reply:
we can always change the content type once we put things in there. I'm just using octet-stream in the current implementation